### DISCUSSION TITLE
Use "flatpak" rather than "xdg" namespace

### DISCUSSION THREAD

**Discussion by probonopd** [flatpak/xdg-desktop-portal#1205] [General]
### Operating System

All

### XDG Desktop Portal version

1.18

### Desktop Environment

Other

### Expected Behavior

The "XDG" namespace is used for specifications under the stewardship of the **Cross-Desktop Group** at https://www.freedesktop.org/.

### Current Behavior

https://github.com/flatpak/ is using the "xdg" namespace as if this was an universally agreed upon standard. Yet it is not a specification listed on https://www.freedesktop.org/wiki/Specifications/.

This might mislead to the wrong impression that this is some kind of of universally accepted standard under the governance of anyone else but the Flatpak team.

### Steps to Reproduce

https://www.freedesktop.org/wiki/

> These specifications mostly cover low-level desktop issues, such as identifying file types, launching applications, and exchanging data between applications and desktops. They are often called 'XDG' specifications, as an acronym for the Cross-Desktop Group.

**Comment by probonopd** [comment]
>  This issue was moved to a discussion. 

Does this mean that this concern is not seen as a valid issue?

**Reply by GeorgesStavracas** [reply]
No. Moving to "Discussion" means it's a discussion, not a bug in code.

**Comment by ebassi** [comment]
The idea that there is such a thing as a "formal" XDG entity is… ahistorical, to say the least. The "cross desktop group" has always been just a bunch of people from different desktop projects agreeing on shared interfaces and publishing them in a shared space. It's also why freedesktop.org does **not** use the term "standard", and prefers "specification". There's no formal process or compliance, and there never was.

Considering that people from GNOME, KDE, and various other desktop projects have agreed on the portal API to the effect of providing implementations, it is all it's needed to fit into the description of "a cross desktop group specification".

**Comment by TingPing** [comment]
This project is also not tied to Flatpak; It is used by Snap and can be used outside of any sandbox as an abstraction between multiple desktops. It does happen to be hosted here but its not like the freedesktop GitLab is active with a diverse group of maintainers.

I'd say it has been extremely successful at bringing a wide range of groups together with [seven desktops](https://github.com/flatpak/xdg-desktop-portal#using-portals) implementing it. Very much in spirit with freedesktop.

**Reply by matthiasclasen** [reply]
I agree. It seems a very difficult argument to make that the portals are not a cross-desktop success story in the spirit of (but much more successful than most) xdg initiatives.

**Reply by probonopd** [reply]
> This project is also not tied to Flatpak

...but seems to be governed by it:

```
https://github.com/flatpak/xdg-desktop-portal
                   ^^^^^^^
```

**Reply by marcthe12** [reply]
I think the reason is that it was originally linked to flatpak in the early days. these day even native unsandboxed app are using portals for certain features. Although probably not worth hassle   it probably could have been moved to freedesktop gitlab or its own org.

**Reply by TingPing** [reply]
This repo is like any open source repo. People who contribute to it get access to it. Just in the past months 2 new maintainers have been added. Neither of those maintainers are active flatpak contributors.

**Reply by matthiasclasen** [reply]
Portals started out as a flatpak technology. They still are. And there is absolutely nothing wrong with that.

**Reply by probonopd** [reply]
What does this project want to imply by using "xdg" as part of the name?

It can be misleading (but at the very least, confusing) to have "xdg" in a project name for a project that is neither an official XDG specification nor has been endorsed by nor is governed by the the Cross-Desktop Group.

If this project implies that "xdg" merely stands for ideas and software that may be used by multiple desktop environments, then AppStream could be named "xdg-appstream", Snap could be renamed "xdg-snap", and AppImage could be named "xdg-appimage" according to the same logic.

**Reply by TingPing** [reply]
I would define it as a project/standard that multiple desktops implement to improve cross-desktop interoperability.

I think appstream fits that definition. Package formats less so.

Flatpak was called `xdg-app` at one point though. I don't agree with that as it has little to do with cross-desktop support beyond using other freedesktop specs as a foundation. The one new foundational technology it created though was `xdg-desktop-portal`. It being split from Flatpak was a general good for the freedesktop ecosystem IMO.

**Reply by TingPing** [reply]
> It can be misleading (but at the very least, confusing) to have "xdg" in a project name for a project that is neither an official XDG specification nor has been endorsed by nor is governed by the the Cross-Desktop Group.

Also this group is less formal than you think. If you look at who has contributed to `xdg-specs` in the past year its a handful of people, many of them `flatpak` contributors. Nobody from desktops would oppose it since they implement it already. I'm sure it would easily be added. Being there is mostly just worse infrastructure though when it comes to documentation/hosting/etc.
